utils/__tests__/check_support_level.test.js

```diff
@@ -0,0 +1,98 @@
+const fs = require('fs');
+const fetch = require('node-fetch');
+const glob = require('glob');
+
+const findSupportLevel = require('../check_support_level');
+
+jest.mock('node-fetch');
+jest.mock('glob');
+jest.mock('fs');
+jest.mock('../helpers', () => ({
+  ...jest.requireActual('../helpers'),
+  checkArgs: jest.fn(),
+}));
+
+global.process.exit = jest.fn();
+global.console.log = jest.fn();
+
+const mockGithubResponse = (result) => {
+  fetch.mockResolvedValueOnce({
+    ok: true,
+    json: jest.fn(() => Promise.resolve(result)),
+    headers: { get: jest.fn() },
+  });
+};
+
+const mockGlobSync = () => glob.sync.mockReturnValueOnce('burrito');
+
+const mockReadFileSync = (supportLevel) => {
+  const yml = `
+name: Taco
+level: ${supportLevel}
+`;
+
+  fs.readFileSync.mockReturnValueOnce(yml);
+};
+
+const STATUS = {
+  ADDED: 'added',
+  MODIFIED: 'modified',
+  REMOVED: 'removed',
+};
+
+describe('Action: Check support level', () => {
```

Comment:
i remember us mentioning multiple support level labels being added. maybe there should be a test for that as well.

Reply:
so true! good idea